packages/client/src/pages/admin/emoji-edit-dialog.vue

```diff
@@ -27,85 +27,75 @@
 </XModalWindow>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script lang="ts" setup>
+import { } from 'vue';
 import XModalWindow from '@/components/ui/modal-window.vue';
 import MkButton from '@/components/ui/button.vue';
 import MkInput from '@/components/form/input.vue';
 import * as os from '@/os';
 import { unique } from '@/scripts/array';
+import { i18n } from '@/i18n';
 
-export default defineComponent({
-	components: {
-		XModalWindow,
-		MkButton,
-		MkInput,
-	},
+const props = defineProps({
+	emoji: {
+		required: true,
+	}
+});
```

Comment:
The codebase usually uses the TypeScript way of `defineProps`:
```suggestion
const props = defineProps<{
	emoji: any,
}>();
```
I'm not quite sure what type `emoji` is so I put `any`. If you know the more specific type that would be appreciated.